lite2/example_test.go

```diff
@@ -57,10 +56,10 @@ func TestExample_Client_AutoUpdate(t *testing.T) {
 			Hash:   header.Hash(),
 		},
 		primary,
-		[]provider.Provider{primary}, // TODO: primary should not be used here
+		[]provider.Provider{primary}, // NOTE: primary should not be used here
 		dbs.New(db, chainID),
-		UpdatePeriod(1*time.Second),
-		Logger(log.TestingLogger()),
+		UpdatePeriod(0), // NOTE: value should be greater than zero
+		// Logger(log.TestingLogger()),
```

Comment:
Why is this commented out?

Reply:
to suppress logger. otherwise you get:

```
got:
I[2020-02-27|16:15:46.267] Starting light client
I[2020-02-27|16:15:48.270] VerifyHeader                                 height=562 hash=007C948D389C14724D28E542E159E7B98ADCB6D8C81DE62655C5EFA6092DEEC3 vals=8E84DAAFA6FFC6CEF14ADA4B34C12D6B7D28B559444EF6A4F96439E349417713
D[2020-02-27|16:15:48.270] Verify newHeader against trustedHeader       trustedHeight=2 trustedHash=30CBED5E0A85457A6F850370899771F9372250AA6DCA0DF7824FB35C90B7A649 newHeight=562 newHash=007C948D389C14724D28E542E159E7B98ADCB6D8C81DE62655C5EFA6092DEEC3
I[2020-02-27|16:15:48.271] Advanced to new state                        height=562 hash=007C948D389C14724D28E542E159E7B98ADCB6D8C81DE62655C5EFA6092DEEC3
successful update
I[2020-02-27|16:15:48.271] Stopping light client
I[2020-02-27|16:15:48.271] Removing all the data
want:
successful update
```